Source/WebCore/platform/graphics/gbm/MemoryMappedGPUBuffer.h

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright (C) 2024 Igalia S.L.
```

Comment:
```suggestion
 * Copyright (C) 2025 Igalia S.L.
```

I first thought that was a typo and this should be 25, but the implementation file says 2024,2025. 

If the file already existed I guess 2024 is OK, but shouldn't header and implementation be aligned?

Reply:
I was not aware that header/impl need to be aligned -- never did that at least :-) I only modify copyright dates in general, if I think I made a significant enough change.

Reply:
However, fixed and set to 2025.